plot_app/configured_plots.py

```diff
@@ -116,13 +116,13 @@ def generate_plots(ulog, px4_ulog, db_data, vehicle_data, link_to_3d_page):
 #        plots.extend(gps_plots)
 
 
-    if is_running_locally():
+#    if is_running_locally():
```

Comment:
Can you remove this commented code?

Reply:
so add the Google map plot when running locally?

Reply:
No, just delete this part.